game_api.txt

```diff
@@ -1124,6 +1124,8 @@ Logs action of the player with a node at a certain position.
 By default only actions of real players are logged.
 Actions of non-players (usually machines) are logged only when
 setting `enable_non_player_action_log` is enabled.
+A player is considered non-player if `player.is_player()` returns
```

Comment:
`player:is_player()`, the dot would trigger an error

Reply:
thx, fixed